functions/src/endpoint_paths/user.ts

```diff
@@ -22,12 +22,11 @@ const cors = require('cors')
 const users_main = express()
 
 users_main.use(users_app)
-users_app.use(express.json())
-users_app.use(express.urlencoded({ extended: true }))
+users_app.use(bodyParser.json())
```

Comment:
Is your local dev branch up to date?  It should not be replacing express with body parser